### PR TITLE
Add wrapup blog post for PDX conference and update website

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+recovery period, and we wanted to send out a quick message to thank
+everyone for helping to make our 1st online conference such a
+success. We're happy to share some of the highlights with you.
+
+Conference recap
+================
+
+At over 600 people this year's conference was our biggest yet! We had
+the usual mix of main stage and lightning talks, unconference sessions,
+writing sprints, and social events. We were very happy how well all our
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/thanks-recap.rst]:
```suggestion
writing sprints, and social events. We were very happy with how well all of our
```

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+events transitioned to our new online format. We also felt the spirit of the community
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/thanks-recap.rst]:
```suggestion
events transitioned to our new online format. We loved how the spirit of the community 
```

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+success. We're happy to share some of the highlights with you.
+
+Conference recap
+================
+
+At over 600 people this year's conference was our biggest yet! We had
+the usual mix of main stage and lightning talks, unconference sessions,
+writing sprints, and social events. We were very happy how well all our
+events transitioned to our new online format. We also felt the spirit of the community 
+come through the screen, which was even more special given the year we've all had.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/thanks-recap.rst]:
```suggestion
was so palpable even in an online event, which was even more special given the year we've all had.
```

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.
+
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened. If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behavior set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+Don’t forget our `Prague conference <https://www.writethedocs.org/conf/prague/2020/>`__ in
+October, and our `Australia+India conference <https://www.writethedocs.org/conf/australia/2020/>`__ in
+December. Tickets are on sale for Prague. The CFP and tickets for the Australia conference are open now.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/thanks-recap.rst]:
```suggestion
December. Tickets are on sale for Prague. The call for talk proposals for the Australia conference is still open, and ticket sales are open.
```

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+Our Slack network has really grown more this last year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, now's the time! Check out the `Write the Docs Slack <http://www.writethedocs.org/slack/>`__.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/thanks-recap.rst]:
```suggestion
Our Slack network has expanded yet again this year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, now's the time! Check out the `Write the Docs Slack <http://www.writethedocs.org/slack/>`__.
```

**Review by plaindocs — COMMENTED** [review]
Looks great! Use or ignore comments as needed.

File: docs/_data/portland-2020-config.yaml [hunk 6]
```diff
@@ -35,13 +35,13 @@ buttons:
 #      link: /livestream
 #     - text: Browse the photos
```
Comment (thatdocslady) [inline docs/_data/portland-2020-config.yaml:36]: We don't have photos from this year :((((((

File: docs/conf/portland/2020/news/thanks-recap.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+Lots of knowledge was shared, and we had a great time.
+
+Videos
+======
+
+Videos of the talks are now available on `our YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ>`__. Sadly, the Monday afternoon talks had an issue with the Q&A not being recorded, so we weren't able to add those to the videos. We think we know what happened with it, and hopefully we've addressed this issue going forward.
+
+Thanks to `Bart at Work <https://www.bartatwork.com/atwork/>`__ for doing such a great job with these. You can also subscribe to our `YouTube channel <https://www.youtube.com/writethedocs>`__ so you don't miss any future content!
+
+
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/thanks-recap.rst:42]:
```suggestion
Shirts still on sale
====================

If you want a souvenir from this year's conference, our `Write the Docs Portland 2020 Pop-Up Shop <https://teespring.com/wtd-portland-2020>`_ is still open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches until August 30th. Shipping is available from the USA or Europe.

```

**Review by thatdocslady — COMMENTED** [review]
Added t-shirts reminder and a question about photos, otherwise LGTM!

File: docs/conf/portland/2020/news/thanks-recap.rst (hunk 4 above)
Comment (mxsasha) [reply docs/conf/portland/2020/news/thanks-recap.rst]: Let's  add something like: 
> Our Prague and Australia+India conferences are also virtual this year, so you can attend or present from anywhere. They take place during the day in the CEST and AEDT time zones, respectively.

**Pull request merged**